.github/workflows/cypress-e2e-test.yml

```diff
@@ -0,0 +1,55 @@
+name: 32.Run E2E Tests
+
+on:
+  workflow_dispatch:
+    inputs:
+      branch:
```

Comment:
We won't need any inputs here; we will get the either branch/tag from the default dropdown option of the dispatcher UI.